railties/test/application/test_runner_test.rb

```diff
@@ -249,6 +252,73 @@ def test_generated_scaffold_works_with_rails_test
       assert_match "0 failures, 0 errors, 0 skips", run_test_command('')
     end
 
+    def test_run_multiple_folders
+      create_test_file :models, 'account'
+      create_test_file :controllers, 'accounts_controller'
+
+      run_test_command('test/models test/controllers').tap do |output|
+        assert_match 'AccountTest', output
+        assert_match 'AccountsControllerTest', output
+        assert_match '2 runs, 2 assertions, 0 failures, 0 errors, 0 skips', output
+      end
+    end
+
+    def test_run_multiple_files_and_run_one_file_by_line_only_runs_filter
```

Comment:
Is this the behavior we want or a limitation by our current integration with minitest?